### PR TITLE
Introducing: Moebius Hostile Envorinment Suit 

### PR DESCRIPTION
**Pull request by Firefox13**
## About The Pull Request

Removes all old anomaly suit mentions and replaces them with the moebius hazard suit and its older brother orange one, which spawns in maint.

## Why It's Good For The Game

The old excavation sprite was terribly outdated, from baystation12 and was a simple spacesuit, not a voidsuit, with separate helmet and suit. Plus it adds yet another (not really terrible actually) voidsuit to maintenance pool.
изображение
изображение

## Changelog
:cl: INFRARED_BARON, Firefox13
del: Excavation suit
add: New moebius hazard suit which spawns everywhere where old excavation suit spawned
add: New hazard suit which spawns in maintenance
/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/void/station.dm [hunk 1]
```diff
@@ -425,4 +425,59 @@
...
+		bomb = 50,
+		bio = 100,
+		rad = 100
+	)
+	siemens_coefficient = 0 // It would be funny get electrocuted to death in a suit that supposed to protect against shocks
+	helmet = /obj/item/clothing/head/space/void/hazardhelmet
+
+
+// hazard suit, moebius version. Only this version actually gives anomalous protection.
+/obj/item/clothing/head/space/void/hazardhelmet/moebius
```
Comment (Kurgis) [inline code/modules/clothing/spacesuits/void/station.dm:470]: This should be contained within the Xenoarch module.

**Comment by Medcar** [comment]
Maybe add it to the tech tree?

File: code/modules/research/xenoarchaeology/artifact/effect.dm [hunk 2]
```diff
@@ -94,10 +94,10 @@ proc/GetAnomalySusceptibility(var/mob/living/carbon/human/H)
 		if(rig.suit_is_deployed() && !rig.offline)
 			protected += 1
 
-	if(istype(H.wear_suit,/obj/item/clothing/suit/space/anomaly))
+	if(istype(H.wear_suit,/obj/item/clothing/suit/space/void/hazardsuit/moebius))
```
Comment (TheShown911) [inline code/modules/research/xenoarchaeology/artifact/effect.dm:97]: Why only the moebius subtype, shouldn't the base hev suit give extra protection too?

File: code/modules/clothing/spacesuits/void/station.dm [hunk 3]
```diff
@@ -425,4 +425,59 @@
...
+	item_state = "hev_violet_helmet"
+	spawn_blacklisted = TRUE
+
+/obj/item/clothing/suit/space/void/hazardsuit/moebius
+	name = "moebius hazard voidsuit"
+	desc = "A moebius branded violet voidsuit that is capable of protecting its user against most hostile environment conditions, including anomalous particles. Integrated self-diagnostics system not included!"
+	icon_state = "hev_violet"
+	item_state = "hev_violet"
+	spawn_blacklisted = TRUE
+	helmet = /obj/item/clothing/head/space/void/hazardhelmet/moebius
```
Comment (TheShown911) [inline code/modules/clothing/spacesuits/void/station.dm:483]: No newline.

**Comment by Medcar** [comment]
cursed pr

**Comment by Firefox13** [comment]
> 
> 
> cursed pr

because of gitkraken refusing to see a new file

**Comment by Firefox13** [comment]
let me fucking redo it

**Pull request closed without merging**